schemas/JSON/manifests/v1.1.0/manifest.installer.1.1.0.json

```diff
@@ -426,6 +558,27 @@
     "RestrictedCapabilities": {
       "$ref": "#/definitions/RestrictedCapabilities"
     },
+    "Markets": {
+      "$ref": "#/definitions/Markets"
+    },
+    "InstallerAbortsTerminal": {
+      "$ref": "#/definitions/InstallerAbortsTerminal"
+    },
+    "ReleaseDate": {
+      "$ref": "#/definitions/ReleaseDate"
+    },
+    "InstallLocationRequired": {
+      "$ref": "#/definitions/InstallLocationRequired"
+    },
+    "RequireExplicitUpgrade": {
+      "$ref": "#/definitions/RequireExplicitUpgrade"
+    },
+    "UnsupportedOSArchitectures": {
+      "$ref": "#/definitions/UnsupportedOSArchitectures"
+    },
+    "AppsAndFeaturesEntries": {
+      "$ref": "#/definitions/AppsAndFeaturesEntries"
+    },
     "Installers": {
       "type": "array",

```

Comment:
Should the length of the installers array be bumped to 1024 here, per https://github.com/microsoft/winget-cli/issues/967?

Reply:
Yes, thanks for catching that!